.vscode/extensions.json

```diff
@@ -0,0 +1,26 @@
+{
+  "recommendations": [
+    // Python
+    "charliermarsh.ruff",
+    "ms-python.python",
+    "ms-python.vscode-pylance",
+    "ms-python.mypy-type-checker",
```

Comment:
Are we adding this solely to force disable it in settings.json?

Reply:
Removed it, isnt needed to disable